<?xml version="1.0"?>
<launch>
    <arg name="audio_direct" default="" /> 
    <arg name="arduino_port" default="" /> 
    
    <node pkg="mesapro" type="audio_alerts.py" name="hri_audio_alerts"> 
	<param name="audio_direct" value="$(arg audio_direct)"/>
    </node>
    <node pkg="rosserial_python" type="serial_node.py" name="rosserial"> 
	<param name="port" value="$(arg arduino_port)" />
    </node>
    <node pkg="mesapro" type="visual_alerts.py" name="hri_visual_alerts"/> 
</launch>


